extensions/graphviz_support/README.md

```diff
@@ -0,0 +1,34 @@
+# Graphviz for LabGraph graphs
+
+Graphviz provide an API to generate a graphviz visualization of the LabGraph topology. 
+
+## Quick Start
+
+### Method 1 - building from source code
+
+**Prerequisites**:
+
```

Comment:
You may want to add the supported python version(s) here.

Reply:
Done